---
title: Human pathogen genomics
search_exclude: true
description: <!---REPLACE THIS with a one sentence description of the page--->
contributors: [Diana Pilvar, Espen Åberg, Wolmar Nyberg Åkerström, Rafael Andrade Buono]
page_id: human_pathogen_genomics
related_pages: 
  your_tasks: [data_brokering, metadata_management, data_transfer]
  tool_assembly: [covid19_data_portal]
  your_domain: [rare_disease_data]
# More information on which page id you can use can be found at https://rdmkit.elixir-europe.org/website_overview
#training:
#  - name:
#    registry:
#    url:
# More information on how to fill in this metadata section can be found here https://rdmkit.elixir-europe.org/page_metadata
---

# Human Pathogen Genomics

## Introduction
The human pathogen genomics domain focuses on studying the genetic code of organisms that cause disease in humans. Studies to identify and understand pathogens are conducted across different types of organisations raging from research institutes to regional public health authorities. And the aims can include urgent outbreak response and prevention measures and developing remedies such as treatments and vaccines.

Data management challenges in this domain include the potential urgency of data sharing and secondary use of data across initiatives emerging from research, public health and policy. And while the pathogenic organisms are the object of interest, there are many consideration to account for when dealing with samples collected from patients, healthcare, and human research subjects.

The genomic data can represent anything from the genetic sequence of a single pathogen isolate to various fragments of genetic materials from a flora of pathogens in larger population. Other data can represent a wide range of contextual information about the human host, the disease, and various environmental factors.

## Planning a study with pathogen genome data

### Description
<blockquote>Sections within Domain pages (aside from "Introduction" at the start and "Relevant tools and resources " at the end) should focus on particular data management problems, which should be described in this first sub-section. For problems that are fully domain-specific, a detailed description is merited. For detailing the domain-specific challenges of a problem that is generic, please link to the corresponding generic Problem page before going into the domain-specific challenges.</blockquote>

While the object of interest in this domain are pathogens, the data is usually derived from samples originating from patients and human research subjects. This means that you must plan to either remove or to handle human data during your study.

### Considerations
<blockquote>Direct and concise considerations, structured in bullet points and typically framed as questions RDMkit reader should ask themselves in order to arrive at the best solution among those listed below. One level of nesting of bullet points within considerations is fine, but more levels should be avoided.</blockquote>

Consant mixing of human and non-human data.

* What legal and ethical aspects do you need to consider? 
  * Can you separate pathogen and human host material and data?
  * What ELSI aspects and contracts with suppliers etc?
  * Who needs to be consulted and how much time would be required to obtain access/permits to get access to samples/data?
* What public health and research initives should you consider aligning with?
  * What data could be shared with or reused from other initiatives during the project?
  * How will you align your practices with convensions from these initiatives to maximise the impact of the data and insight generated by the project? 
  * How will you share data with your collaborators and other initiatives?
* What conventions will you adopt when planning your study?
  * What existing protocols should you consider adopting for sample preparation, sequensing, variant calling etc? 
  * What conventions should you adopt for documenting your research? 

### Solutions
<blockquote>Detail, either in normal text or in bullet points, the domain-specific solutions to the problem. Do not merely list tools or resources, as they will be automatically listed in the bottom section, but you can and should mention tools and resources listed below if you detail their usage to solve the problem. </blockquote>

#### Isolate pathogen from host information
* Depending on the pathogen, how it interacts with the host, or the methods applied, it can be possible to generate clean isolates that do not contain host related material. Data produced from a clean isolate could potentially be handled with few restrictions, while other data will be considered to be personal and [sensitive](sensitive_data) that need [protection](data_protection)

#### Public health initiaitves
* National and international recommendations from public health authorities, epidemic surveillance programs and research data communities should be considered when planning a new study or surveillance programme. In particular, you could consult relevant guidance issued by national and international surveillance programs while considering widely adopted guidelines for research documentation, and recommendations provided by data sharing platforms and communities.
  * [WHO genomic surveillance strategy](https://www.who.int/initiatives/genomic-surveillance-strategy)
  * [European Centre for Disease Prevention and Control](https://www.ecdc.europa.eu/en)
  * [ECDC’s Surveillance and study protocols](https://www.ecdc.europa.eu/en/covid-19/surveillance/study-protocols)
  * [Centers for Disease Control and Prevention (CDC)](https://www.cdc.gov/genomics/pathogen/index.htm)
  * [NCBI Pathogen Detection](https://www.ncbi.nlm.nih.gov/pathogens/)
  * [EMBL-EBI Pathogens](https://www.ebi.ac.uk/ena/pathogens/home)
  * [National resources \| RDMkit](national_resources)

#### Documenting sequencing experiments
* Good practices for annotating sequencing experiments suggest that the documentation, at a minimum, should describe the design of the study or surveillance program, the collected specimens and how the samples were prepared, the experimental setup and protocols, and the analysis workflow.

  * [Minimal Information about a high throughput SEQuencing Experiment (MINSEQE)](https://fairsharing.org/FAIRsharing.a55z32)
  * The Genomic Standards Consortium (GSC) have defined a set of core and extended descriptors, the [Minimum Information about any Sequence (MIxS)](https://fairsharing.org/FAIRsharing.9aa0zp), for genomes and metagenomes with associated samples and their environment to guide scientists on how to capture the metadata essential for high quality research. 
  * The RDMkit page on [documentation and metadata](Documentation_and_metadata) provides a guide to how you can document data during a project.
  * This paper give [Ten simple rules for annotating sequencing experiments](https://doi.org/10.1371/journal.pcbi.1008260) to help data producers collect and store high-quality metadata about sequencing experiments.
  * ISO, the International Organization for Standardization have several general guidelines relevant for both the laboratory and bioinformatic components of genome sequencing, i.e.
[ISO 20397-1:2022](https://www.iso.org/standard/74054.html),
[ISO/TS 20428:2017](https://www.iso.org/standard/67981.html),
[ISO 20397-2:2021](https://www.iso.org/standard/67895.html),
[ISO/TS 20428:2017](https://www.iso.org/obp/ui/#iso:std:iso:ts:20428:ed-1:v1:en),
[ISO/TS 22692:2020](https://www.iso.org/standard/73693.html),
[ISO 23418:2022](https://www.iso.org/obp/ui/#iso:std:iso:23418:ed-1:v1:en). (NB! requires a subscription or one time purchase)

## Collecting and processing pathogen sequence data

### Description
<blockquote>Sections within Domain pages (aside from "Introduction" at the start and "Relevant tools and resources " at the end) should focus on particular data management problems, which should be described in this first sub-section. For problems that are fully domain-specific, a detailed description is merited. For detailing the domain-specific challenges of a problem that is generic, please link to the corresponding generic Problem page before going into the domain-specific challenges.</blockquote>

### Considerations
<blockquote>Direct and concise considerations, structured in bullet points and typically framed as questions RDMkit reader should ask themselves in order to arrive at the best solution among those listed below. One level of nesting of bullet points within considerations is fine, but more levels should be avoided.</blockquote>

* What information should you consider recording when collecting data?
  * What should you note when collecting, storing and preparing the samples?
  * How will you capture information about the configuration and quality of the sequencing results?
  * How will you ensure that the information captured is complete and correct?
* What data and file formats should you consider for your project?
  * What are the de-facto standards used for the experiemnt type and down-stream analysis-pipelines?
  * Where are the instrument specific aspects for the data and files formats documented?
* What existing data will you integrate or use as a reference in  your project?
  * What reference genome(s) will you need access to?
  * What is the recommended citation for the data and their versions?

### Solutions
<blockquote>Detail, either in normal text or in bullet points, the domain-specific solutions to the problem. Do not merely list tools or resources, as they will be automatically listed in the bottom section, but you can and should mention tools and resources listed below if you detail their usage to solve the problem. </blockquote>

#### Filtering genomic reads coresponding to human dna fragments

* Data files with reads produced by sequencing experiments sometimes contain fragments of the host organism’s DNA. When the host is a human research subject or patient, these fragments can be masked or removed to produce files that could potentially be handled with fewer restrictions.
  * Mapping to human reference genomes
  * Mapping to pathogen genome

#### Contextual information about the sample

* Information about the host phenotype, context and disease is often necessary to answer questions in a research study or policy perspective. This information can contain personal and sensitive data.
  * Pathogens checklist(s) can be found among [ENA Sample checklists](https://www.ebi.ac.uk/ena/browser/checklists), including checklists derived from the [MIxS consortium](http://w3id.org/mixs). For example the [ENA virus pathogen reporting standard checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000033) has been recurrently used for SARS-CoV-2 studies.
 
<!--- Other potential checklist examples
   * [ENA prokaryotic pathogen minimal sample checklist](https://www.ebi.ac.uk/ena/browser/view/ERC000028)
* [ENA binned metagenome](https://www.ebi.ac.uk/ena/browser/view/ERC000050)
    * [ENA Global Microbial Identifier reporting standard checklist GMI_MDM:1.1](https://www.ebi.ac.uk/ena/browser/view/ERC000029)
    * [GSC MIxS host associated](https://www.ebi.ac.uk/ena/browser/view/ERC000013)
    * [GSC MIxS human associated](https://www.ebi.ac.uk/ena/browser/view/ERC000014)
    * [GSC MIxS microbial mat biolfilm](https://www.ebi.ac.uk/ena/browser/view/ERC000019)
    * [GSC MIxS human gut](https://www.ebi.ac.uk/ena/browser/view/ERC000015)
    * [GSC MIxS human oral](https://www.ebi.ac.uk/ena/browser/view/ERC000016)
    * [GSC MIxS human skin](https://www.ebi.ac.uk/ena/browser/view/ERC000017)
    * [GSC MIxS human vaginal](https://www.ebi.ac.uk/ena/browser/view/ERC000018)--->

* Other contextual information can include non-host related environmental factors, such as interactions with other pathogens, drugs and geographic proliferation. This information can be used for 
  * Ontologies and checklists
      * [Phenotypic QualiTy Ontology](https://bioportal.bioontology.org/ontologies/PATO)
      * [NCBI Taxonomy](https://www.ncbi.nlm.nih.gov/taxonomy)
      * [Disease Ontology](https://disease-ontology.org)
      * [Chemical Entities of Biological Interest](https://bioportal.bioontology.org/ontologies/CHEBI/?p=summary)
      * [UBER anatomy ONtology](https://bioportal.bioontology.org/ontologies/UBERON)

  * Pathogens checklist(s) for ENA/packages for SRA? Etc.?

* Information about the sampled material and how it was processed for sequencing

#### Genomic information and quality assessment
* Make sure you are using common and suitable data formats
  * Enough pathogen materials?(https://www.sequencing.uio.no/pacbio-services/dna-requirements/) Was it stored properly?
  * Repositories generally have information about [data formats](Data_publication)
  * [High-Throughput Sequencing \| LifeScienceRDMLookUp](https://elixir.no/rdm-lookup/sequencing)
  * 

* Example of a common pipeline structure and corresponding outs with and example of quality information...

* Checklists and convenstion for describing experiment configuration and execution
  * checklists

* Data quality assurance  
NGS pipelines are comprised of several elements, all of which contribute to the end quality of the result, from the reception of the samples to delivery of the outcomes. For this reason, quality control (QC) steps should be incorporated into the workflow to ensure that the data is fit for use, and its usage poses no risk to the patient. 
    * [The Beyond One Million Genomes (B1MG)](https://b1mg-project.eu) project provides guidelines that cover the minimum [quality requirements](https://zenodo.org/record/5018495) for the generation of genome sequencing data.
    * A good place to look for scientific and technical information about data quality validation software tools for pathogenomics is  [Bio.Tools](https://bio.tools/t?page=1&q=validation&sort=score&topicID=%22topic_3168%22).
  * The [FAIR Cookbook](https://faircookbook.elixir-europe.org/content/home.html) provides instructions on validation of [file formats](https://faircookbook.elixir-europe.org/content/recipes/interoperability/fastq-file-format-validators.html)
  * The Galaxy Training Network provides free on-line [training material](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial.html) on quality control.


#### Describing samples and processing steps

* Separating pathogen from human
* Sharing/Public
* Dealing with patient/research subject information
* Analysis
* Workflow harmonization
* Link to Galaxy showcase in IDTk
* Automated pipelines
* Collections: Galaxy workflows, WorkflowHub, other sources?

## Sharing and preserving pathogen genomic data


### Description
<blockquote>Sections within Domain pages (aside from "Introduction" at the start and "Relevant tools and resources " at the end) should focus on particular data management problems, which should be described in this first sub-section. For problems that are fully domain-specific, a detailed description is merited. For detailing the domain-specific challenges of a problem that is generic, please link to the corresponding generic Problem page before going into the domain-specific challenges.</blockquote>

### Considerations
<blockquote>Direct and concise considerations, structured in bullet points and typically framed as questions RDMkit reader should ask themselves in order to arrive at the best solution among those listed below. One level of nesting of bullet points within considerations is fine, but more levels should be avoided.</blockquote>

* What data need to be preserved by the project and for how long?
  * What is preserved by others and how would someone find and access the data?
* What databases should I use to share human pathogen genomics data?


### Solutions
<blockquote>Detail, either in normal text or in bullet points, the domain-specific solutions to the problem. Do not merely list tools or resources, as they will be automatically listed in the bottom section, but you can and should mention tools and resources listed below if you detail their usage to solve the problem. </blockquote>

#### Sharing host related and other contextual information
* Something about links to non-sensitive pathogen genome data etc. Aggregate level information, data masking and access to detailed records. Reference human data. Some of this information can be used as metadata for genomic data. 

#### Sharing pathogen genomic data
Several platforms can be used in parallel to reach different audiences and include national and European healthcare surveillance systems administered by public health authorities, such as the ECDC’s TESSy/EpiPulse; international research data exchanges such as INSDC (EMBL-EBI, NCBI, DDBJ) and the Federated EGA network; and virus specific initiatives such as GISAID. 

* Something about INSDC, GISAID, EBI-data base chooser etc. also FEGA, ECDC, national data hubs etc.
  * EBI Pathogens data hubs
  * Differences between Health and Research
  * Data ownership
  * [Data brokering](https://rdmkit.elixir-europe.org/data_brokering)
  * Data sharing agreements/conditions
  * Platforms for sharing

